src/connection.cpp

```diff
@@ -26,16 +26,13 @@
 #include "scheduler.h"
 #include "server.h"
 
-bool Connection::logError = true;
-
 extern ConfigManager g_config;
 
-Connection_ptr ConnectionManager::createConnection(boost::asio::ip::tcp::socket* socket,
-        boost::asio::io_service& io_service, ServicePort_ptr servicer)
+Connection_ptr ConnectionManager::createConnection(boost::asio::io_service& io_service, ConstServicePort_ptr servicePort)
 {
 	std::lock_guard<std::mutex> lockClass(connectionManagerLock);
-
-	auto connection = std::make_shared<Connection>(socket, io_service, servicer);
+	
```

Comment:
That's the only thing I hate in my current IDE... it leaves trailing tabs (spaces are removed correctly most of the time).